docs/user/resources/04-40-10-azure-vpc-peering.md

```diff
@@ -5,20 +5,26 @@ The `azurevpcpeering.cloud-resources.kyma-project.io` custom resource (CR) speci
 Kyma and the remote Azure Virtual Private Cloud (VPC) network. Virtual network peering is only possible within the networks
 of the same cloud provider.
 
-Once an AzureVpcPeering CR is created and reconciled, the Cloud Manager controller first creates a virtual network peering 
-connection in the Virtual Private Cloud (VPC) network of the Kyma cluster in the underlying cloud provider and accepts
-VPC peering connection in the remote cloud provider subscription.
+Once an `AzureVpcPeering` CR is created and reconciled, the Cloud Manager controller creates a VPC peering connection in
+the VPC network of the Kyma cluster in the underlying cloud provider subscription and accepts VPC peering connection in 
```

Comment:
```suggestion
the VPC network of the Kyma cluster in the underlying cloud provider subscription, and accepts a VPC peering connection in 
```
I think "connection" needs the "a", but let me know if I misunderstand the sentence